### DISCUSSION TITLE
Chia plotting results in "Error 1"

### DISCUSSION THREAD

**Discussion by Dananddog** [Chia-Network/chia-blockchain#4860] [Support]
Hi Github, 
Thanks for the repository of knowledge you've provided me over time. I've run into one that googling hasn't helped. 

I'm plotting for the Chia blockchain. I'm on a mid 2010 Mac Pro with a dual boot to run Ubuntu 20.04 since I couldn't upgrade the mac without buying a graphics card. 
The Mac is a dual Xeon 6c/12t with 96gb ram, a nvme 500gb ssd, 1tb hdd (boot) and 8tb hdd (external). 

The problem I'm running into is "error 1" in the log of the plotting process. It happens, seemingly every time at each step at bucket 37, and will delay the process to the extent that my old core 2 duo machine is actually plotting about twice as fast... with only 6gb of ram, meaning I can only run one plot at a time. 

In searching, the error is usually caused by a lack of storage, or a lack of ram. I re-formatted the nvme to FAT32 (so I can move the plots to a windows machine to run it later) and verified that I have 460 GiB available. 
Next, I verified that the external drive has 6.6 TB available. 
Finally, I looked in the terminal at sudo top -i and saw that the process is indeed only taking up about 3GiB Ram, and averaging 10-15% cpu usage. 

This has occurred plotting in parallel as well as serial, assigning anywhere from 2400 MiB to 3390Mib Ram, and assigning anywhere from 2-4 threads to each plot. 
![Screenshot from 2021-05-13 14-08-15](https://user-images.githubusercontent.com/84155471/118188059-b2515980-b3f4-11eb-8311-d0e724e8ed57.png)

I'm at a loss, can anyone help?

**Comment by JoshCork** [comment]
I'm having the same issue.  Confirmed it's not an issue with my NVME plotter as I've swapped back and forth between my MacBook Pro and am only having these issues on my Ubuntu 20.04 machine running the GUI.  I haven't yet tried the CLI.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
